src/Symfony/Component/Form/FormBuilderInterface.php

```diff
@@ -15,6 +15,8 @@
  * @author Bernhard Schussek <bschussek@gmail.com>
  *
  * @extends \Traversable<string, FormBuilderInterface>
+ * 
+ * @method bool isSubmittedRequestValid(Request $request) Call {@link handleRequest()} and return the result of the condition {@link isSubmitted()} and {@link isValid()}
```

Comment:
this should be defined over the `FormInterface` instead, right?

Reply:
Yes, sorry